feature-group-definitions/has.dist.yml

```diff
@@ -2,6 +2,7 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 name: ":has()"
+description: The `:has()` CSS pseudo-class selects an element if any of the selectors passed as parameters would match at least one element.
```

Comment:
The word selector feels a bit confusing to me. I always had it in my mind that selector meant "the entire thing that's to the left of the `{` character". But I understand that some parts of what I call a selector can also be called selectors.

From https://www.w3.org/TR/selectors/#structure:
> The term [selector](https://www.w3.org/TR/selectors/#selector) can refer to a [simple selector](https://www.w3.org/TR/selectors/#simple), [compound selector](https://www.w3.org/TR/selectors/#compound), [complex selector](https://www.w3.org/TR/selectors/#complex), or [selector list](https://www.w3.org/TR/selectors/#selector-list).

So, said differently, the word "selector" can be used for any part of what's to the left of `{` and for the entire thing too.

With that in mind, I would probably stay away from using that word unless there's no possible confusion as to what it means. I would vote for either pseudo-class function or functional pseudo-class.

Reply:
Either "pseudo-class function" or "functional pseudo-class" would be OK with me. MDN uses both in different places:
https://developer.mozilla.org/en-US/docs/Web/CSS/:is
https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes

I think I prefer "functional pseudo-class" because it's a modifier to "pseudo-class" similar to "global method". And "functional pseudo-class" gets about twice as many hits on Google. Let's go with that for now?